lib/kflags/kconfig/config.go

```diff
@@ -281,18 +283,45 @@ func FromFlags(fl *Flags) Modifier {
 	}
 }
 
+func FallbackConfig(cs cache.Store) (config.Store, error) {
+	dir, _, err := cs.Get("enkit-config-discovery://")
+	if err != nil {
+		return nil, err
+	}
+	dir, err = cs.Commit(dir)
+	if err != nil {
+		return nil, err
+	}
+	store, err := directory.OpenDir(dir)
```

Comment:
I would just use the marshal/unmarshal function, skipt the config.Store layer entirely, with https://pkg.go.dev/github.com/enfabrica/enkit/lib/config/marshal#MarshalFile?

Reply:
constant file name, fallback.yaml or ... with the key changing per domain?

Reply:
Have pushed this. It doesn't improve the code much, in fact arguably the manual path manipulation is uglier (and not portable, will push another fix using os.PathSeparator in a minute), but it does remove some imports.